catalog/config.json

```
@@ -0,0 +1,17 @@
+{
+    "tagline": "An interactive funnel depicting your customers’ journey",
```

This tagline is a bit too long and will fail validation (max is 30 character). Here are the validation guidelines: https://docs.google.com/document/d/1IsDJ2VH_pnwImXnYOPIf08kyv0m93itVKrC8-RQXT4U/edit

```suggestion
    "tagline": "An interactive NRQL funnel",
```